models/node.go

```diff
@@ -118,3 +118,7 @@ type ChainDetails struct {
 	NetworkID       uint64
 	TotalDifficulty *big.Int
 }
+
+func (d *ChainDetails) IsEmpty() bool {
+	return d.NetworkID == uint64(0)
```

Comment:
I know it is very simple, but it might be a good idea to have a constant for an empty chain